packages/nx/src/command-line/migrate.ts

```diff
@@ -906,7 +911,34 @@ async function runMigrations(
         await import('../adapter/ngcli-adapter')
       ).runMigration(root, m.package, m.name, isVerbose);
     }
+
     logger.info(`Successfully finished ${m.name}`);
+
+    if (shouldCreateCommits) {
+      const commitMessage = `chore: [nx migration] ${m.name}`;
```

Comment:
What would you think about parameterizing at least the 'chore: ' portion of the commit message? Adding something like '--commit-prefix' as an example? 

Many commitlint setups require a scope.